app/models/manual.rb

```diff
@@ -28,6 +28,10 @@ def base_path
     end
   end
 
+  def published?
+    live? || redrafted?
+  end
```

Comment:
We added support for the `redrafted` status to Publishing API because we thought there was some requirement in Specialist Publisher for this information. Given this seems to be abstracting over those two states anyway, is it really needed? (cc @danielroseman)

I had a quick look for 'redrafted' in specialist publisher and this seems to be the only usage?